src/Lumi/Styles/Theme.purs

```diff
@@ -24,3 +56,14 @@ lumiThemeContext =
 
 useTheme :: Hook (UseContext LumiTheme) LumiTheme
 useTheme = useContext lumiThemeContext
+
+textFontSize :: LumiTheme -> (forall a. TextMap a -> a) -> Int
+textFontSize (LumiTheme { fontSizes }) selector = selector fontSizes
+
+textLineHeight :: LumiTheme -> (forall a. TextMap a -> a) -> Int
+textLineHeight (LumiTheme { fontSizes, lineHeightFactor }) selector =
+  Int.floor $ Int.toNumber (selector fontSizes) * lineHeightFactor
+
+textMargin :: LumiTheme -> (forall a. TextMap a -> a) -> Int
+textMargin (LumiTheme { fontSizes, textMarginFactor }) selector =
+  Int.floor $ Int.toNumber (selector fontSizes) * textMarginFactor
```

Comment:
The forall here seems a little strange since `LumiTheme` already constrains it to `Int`. If the `LumiTheme` type ever changed this would probably be a confusing error. Maybe there's an advantage to it I don't understand though. Not a big deal.

Reply:
This forall just means that it expects a function that can select the contents of `TextMap`, so if the type of `LumiTheme` ever changed this wouldn't produce any errors, unless we removed `fontSizes :: TextMap Int` from it.